docs/guide/algob-config.md

```diff
@@ -33,55 +33,61 @@ loggingEnabled?: boolean;
 initialDate?: string;
 ```
 
-## Credentials
+## Network Credentials
 
 If you are running a local node, you can find port and token values for your algod in `$ALGORAND_DATA/algod.net` and `$ALGORAND_DATA/algod.token` files.
 
-You can specify **Algod** Credentials in network object or you can load it from ENV.
-To load it from ENV:
+You can specify **Algod** [Network Credentials](https://algobuilder.dev/api/algob/interfaces/types.NetworkCredentials.html) manually or you can load it from ENV.
 
-- Method 1
+- Method 1: Specify manually:
 
-  - To add `ALGOD_ADDR` and `ALGOD_TOKEN` in env, you can use the following commands in terminal:
+        let algodCred = {host: "127.0.0.1", port: 4001, token: "c3e..."}
+
+- Method 2
```

Comment:
Can we also give this method also a name similar to method 1?

Reply:
OK